src/api/entities/Identity/index.ts

```diff
@@ -691,21 +705,45 @@ export class Identity extends Entity<UniqueIdentifiers, string> {
       },
     } = this;
 
-    const assembleResult = ({
-      secondaryKeys: secondaryAccounts,
-    }: PolymeshPrimitivesIdentity): PermissionedAccount[] => {
-      return secondaryAccounts.map(({ signer: rawSigner, permissions }) => ({
-        account: signatoryToAccount(rawSigner, context),
-        permissions: meshPermissionsToPermissions(permissions, context),
-      }));
+    const assembleResult = (
+      rawSecondaryKeyKeyRecord: Option<PolymeshPrimitivesSecondaryKeyKeyRecord>[],
+      accountIds: AccountId32[]
+    ): PermissionedAccount[] => {
+      return rawSecondaryKeyKeyRecord.reduce((answer, optKeyRecord, index) => {
+        if (optKeyRecord.isSome) {
+          const account = accountIdToAccount(accountIds[index], context);
+          const keyRecord = optKeyRecord.unwrap();
+
+          if (keyRecord.isSecondaryKey) {
+            const [, permissions] = keyRecord.asSecondaryKey;
+            answer.push({
+              account,
+              permissions: meshPermissionsToPermissions(permissions, context),
+            });
+          }
+        }
+
+        return answer;
+      }, [] as PermissionedAccount[]);
     };
 
+    // TODO should use pagination
```

Comment:
https://polymath.atlassian.net/browse/DA-341

I think its OK to merge in to alpha without pagination, but it should be added in since 200 is no longer the limit before beta.